book/aliases.md

```diff
@@ -30,4 +30,4 @@ Your useable aliases can be seen in `$nu.scope.aliases`.
 
 To make your alias persistent it must be added to your _config.nu_ file.
 
-For more details about how to persist aliases so that they're visible when you start up Nushell, see the [configuration chapter](configuration.md).
+Just put them in your _config.nu_ by running `config nu` and `exec nu` to reload your nushell.
```

Comment:
`exec nu` is only a linux/mac thing and not cross platform. I prefer it to end with ` and restart nushell` or something like that.

Reply:
sorry haven't used windows in decades - so forgot that - I'll take care of that!

Reply:
ah and I saw there is a german version that I forgot!